modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/RuleMatcher.php

```diff
@@ -91,14 +91,14 @@ public function __construct(AliasManagerInterface $aliasManager, CacheBackendInt
   /**
    * {@inheritdoc}
    */
-  public function getMatchingSection(string $path = NULL): ?string {
```

Comment:
I am aware that these changes were already introduced by @22Alexandra in EWPP-4782, but I put it in the separate commit, so it is easier to rebase.